src/EPiServer.SocialAlloy.ExtensionData/Membership/MembershipModeration.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using EPiServer.Social.Groups.Core;
 
-namespace EPiServer.SocialAlloy.Web.Social.Models.Moderation
+namespace EPiServer.SocialAlloy.ExtensionData.Membership
```

Comment:
If I understand your suggestion you are recommending that we have the namespace for the extensiondata proj be EPiServer.SocialAlloy.Web.ExtensionData? Is that correct? That would be more consistent I suppose. I am a little confused as to why the ".Web" was really necessary in the first place for the alloy proj. Any idea?